packages/manager/modules/pci/src/projects/project/serving/namespace/models/service/model.class.js

```diff
@@ -0,0 +1,43 @@
+import {
+  PROCESSING_VERSION_STATUS, ERROR_VERSION_STATUS, VERSION_STATUS, API_STATUS,
+} from '../models.constants';
+
+export default class Model {
```

Comment:
```suggestion
export default class NamespaceModel {
```
would be better. `Model` is too generic